homebrew_challenge/src/components/Facts/Facts.css

```diff
@@ -0,0 +1,71 @@
+.icons-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.facts-container {
+  margin: 0;
+  padding: 20px;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  font-size: 14px;
+  background-color: #f2f2f2;
+}
+
+.facts-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.facts-component {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  text-align: left;
+  width: 19%;
+  min-height: 110px;
+  padding: 10px;
+  box-shadow: 10px 10px 5px #aaaaaa;
+  background-color: #ffffff;
+}
+
+.facts-icon {
+  padding: 5px;
+  background-color: #ffffff;
+  height: 6rem;
```

Comment:
Height/width needs to be at most 5rem to fit in the boxes at the narrowest browser width before the section vanishes altogether, and I think 4rem looks fine.

Reply:
I've reduced to 5rem and taken out padding-top that was extra.